letsdrill.rb

```diff
@@ -1,24 +1,33 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if (integer >= 90)
+    "A"
+  elsif (integer >= 80)
+    "B"
+  elsif (integer >= 70)
+    "C"
+  elsif (integer >= 60)
+    "D"
+  else
+    "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.inject do |memo, arrayElement|
```

Comment:
I like the usage of `inject` here, but ruby has a built-in for this kind of operation